src/variableinspector.ts

```diff
@@ -8,12 +8,25 @@ import { DockLayout, Widget } from '@lumino/widgets';
 
 import { IVariableInspector } from './tokens';
 
+import {
+  DataGrid as WebDataGrid,
+  DataGridRow,
+  DataGridCell,
+  allComponents,
+  provideJupyterDesignSystem,
+  Select,
+  Option,
+  Search,
+  Button
+} from '@jupyter/web-components';
+provideJupyterDesignSystem().register(allComponents);
```

Comment:
You should import the `jp...` variant of the components.

Then could import as fine grain the proper components to optimize what is loaded:

```suggestion
provideJupyterDesignSystem().register(jpDataGrid(), jpDataGridRow(), ...);
```